src/model/custom_electra/electra.py

```diff
@@ -0,0 +1,114 @@
+from typing import Optional
+import torch
+import transformers
+from transformers import ElectraForMaskedLM, ElectraForPreTraining 
+from transformers import AutoConfig, PretrainedConfig, PreTrainedModel, AutoModelForMaskedLM
+#from transformers.tokenization_utils import PretrainedTokenizerFast
+from transformers.models.electra.modeling_electra import ElectraForPreTrainingOutput
+
+
+
+class CustomElectraConfig(PretrainedConfig):
+    model_type= "custom_electra"
+
+    def __init__(
+        self,
+        generator_model: transformers.DebertaV2Config,
+        discriminator_model: transformers.DebertaV2Config,
+        vocab_size: int,
+        embedding_sharing: bool= True,
+        discriminator_lambda_loss: float= 50.0, 
+        pretrained_model_name_or_path= None,
+        **kwargs,
+    ):  
+        print("===== kwargs =====")
+        for key, value in kwargs.items():
+            print(f"{key}: {value}")
+        print(f'pretraineds_model_name_or_path: {pretrained_model_name_or_path}')
+        #super().__init__(pretrained_model_name_or_path)
+        super().__init__(**kwargs)
+        self.vocab_size= vocab_size
+        self.generator_config= generator_model
```

Comment:
Should this be `self.generator_model` instead of `.generator_config` (and same for the discriminator below?

Reply:
Actually, I think the variables in the `kwargs` should just be named `_config` instead of the `_model` as they are now